lib/src/barback/dartdevc/dartdevc.dart

```diff
@@ -65,40 +65,46 @@ Map<AssetId, Future<Asset>> bootstrapDartDevcEntrypoint(
   }
 
   () async {
-    var module = await moduleReader.moduleFor(dartEntrypointId);
+    try {
+      var module = await moduleReader.moduleFor(dartEntrypointId);
+      if (module == null) {
+        outputCompleters.values.forEach((c) =>
```

Comment:
[nit] consider using for loops rather than forEach throughout

https://www.dartlang.org/guides/language/effective-dart/usage#avoid-using-iterableforeach-with-a-function-literal